lib/github.js

```diff
@@ -29,6 +29,12 @@ const closePullRequest = async (context, pullRequestNumber) => {
   );
 };
 
+const mergePullRequest = async (context, pullRequestNumber) => {
+  await context.github.pullRequests.merge(
```

Comment:
I believe the `await` keyword is un-needed here if you `return` this promise - because this function is `async` this function will only trigger its Promise once all promises return from inside

Reply:
returning the result of the merg may also help with error logging - in the case this fails in some unexpected case!